packages/react-native-lockdown/src/lockdown-serializer.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Types for `lockdownSerializer()`
+ *
+ * @packageDocumentation
+ */
+
+/**
+ * Input serializer configuration for `lockdownSerializer()`.
+ *
+ * @privateRemarks
+ * This should be the same type as `metro-config`'s `SerializerConfigT`; we only
+ * need a tiny slice of types from `metro-config` and it is not worth it to add
+ * as a production dependency.
+ * @see {@link https://github.com/facebook/metro/blob/main/packages/metro-config/types/configTypes.d.ts}
+ */
+export interface LockdownSerializerConfig {
+  getPolyfills?: GetPolyfillsFn
+
+  getRunModuleStatement?: GetRunModuleStatementFn
+
+  [key: PropertyKey]: unknown
+}
```

Comment:
i was hoping we could follow convention
https://github.com/react-native-community/template/blob/main/template/metro.config.js#L7
so import `'@react-native/metro-config').MetroConfig['serializer']`
so we maintain less types

then since it's a devDependency in typical RN project
https://github.com/react-native-community/template/blob/main/template/package.json#L26
we add it as a peerDependency

unless you still reckon not worth the bother 😜